client/src/main/java/com/paypal/selion/reports/runtime/Gatherer.java

```diff
@@ -41,7 +44,10 @@ static String saveGetLocation(WebDriver driver) {
                 location = driver.getCurrentUrl();
             }
         } catch (Exception exception) {
-            logger.log(Level.WARNING, "Current location couldn't be retrieved by getCurrentUrl().", exception);
+            logger.log(
```

Comment:
@ILikeToNguyen 

>  the only place i think that actually reads this 'location' value in BaseLog is from HtmlReporter, the old report. So might be worth just logging it to finer since this is our old report. Or perhaps we need to add this to our new Reporter since it was in our old report...

I suspect we probably do need to add it to the new report.

That said, we still need to do something better here. This is the one part of this PR that is keeping me from merging.. I'm not super happy seeing this message on the console. Also, if the exception is caught for a non-appium case I now get this additional text which, to me is now noise (for the non-appium case) -- `This can be SAFELY IGNORED if testing a non-web mobile application` 

How about checking the exception message in the `catch` block for `Not yet implemented ... "appium.io"` and if it exists, log a different message like you are now, but maybe more concise.
```java
logger.log(Level.WARNING, "Appium does not support getCurrentUrl(). This warning can be safely ignored.");
```
Otherwise, log the message as before with 
```java 
logger.log(Level.WARNING, "Current location couldn't be retrieved by getCurrentUrl().", exception);
```

@krmahadevan @fakrudeen78 @sebady do any of you have a better suggestion?

All - do any other drivers hit this `catch` block when `getCurrentUrl()` is called?